internal/command/deploy/deploy.go

```diff
@@ -163,12 +163,16 @@ func DeployWithConfig(ctx context.Context, appConfig *app.Config, args DeployWit
 		if err != nil {
 			return fmt.Errorf("error loading appv2 config: %w", err)
 		}
+		region := appConfig.PrimaryRegion
```

Comment:
Does this change what fly.toml config that is produced when launching nomad apps? For example, will this start adding a primary_region field to the fly.toml?

We probably don't want that for nomad, just in case that changes behavior somewhere else.